src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -29,6 +29,7 @@
 #include <set>
 
 #include "absl/strings/string_view.h"
+#include "third_party/absl/strings/str_cat.h"
```

Comment:
Need to remove "third_party/" from this include.

Reply:
Whoops! Bad copy/paste, fixed